aria/orchestrator/workflows/builtin/utils.py

```diff
@@ -68,3 +67,11 @@ def get_task(node_instance_id):
                     graph.add_dependency(dependency, task)
             else:
                 graph.add_dependency(task, dependencies)
+
+
+def _has_operation(interfaces, operation_name):
+    for interface in interfaces:
+        for operation in interface.operations:
```

Comment:
I was not aware of that API. By the way, is there a "has_a" API just to check existence? That would be enough.